keentools/facetracker/interface/panels.py

```diff
@@ -800,6 +803,11 @@ def _mask_2d_block(self, layout: Any, geotracker: Any,
         row.prop(geotracker, 'mask_2d_inverted',
                  text='', icon='ARROW_LEFTRIGHT')
 
+        if geotracker and geotracker.mask_2d:
+            col2 = col.column(align=True)
+            col2.active = True
+            col2.prop(geotracker.mask_2d, 'frame_start')
+
         row = layout.row(align=True)
```

Comment:
shouldn't be updated according to new layout?

Reply:
Everything is ok, it's regular pattern to make UI